test/with_dummyserver/test_connectionpool.py

```diff
@@ -772,6 +772,13 @@ def test_mixed_case_hostname(self):
         response = pool.request('GET', "http://LoCaLhOsT:%d/" % self.port)
         self.assertEqual(response.status, 200)
 
+    def test_broken_pipe_ignore(self):
```

Comment:
Let's split this testcase into two testcases one with the default `self.pool.urlopen()` which defaults to unchunked and one that actually specifies `chunked=True` but for the chunked testcase we need to skip on Google App Engine because it doesn't support chunked requests in URLFetch.  You might have to create a new decorator for `notAppEngine` within `test/__init__.py` to do this.

Reply:
You could maybe try skipping if `urllib3.contrib.appengine.is_appengine()` is `True`?

Reply:
I tried to push the change but you already pushed it haha. Thanks!!!